rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/SelectionRanges.java

```diff
@@ -53,23 +53,39 @@ private SelectionRanges() { /* hide implicit constructor */ }
      * @param columns The editor's column map.
      * @return A range with optional parent ranges.
      */
-    public static @Nullable SelectionRange toSelectionRange(IList ranges, ColumnMaps columns) {
+    public static SelectionRange toSelectionRange(IList ranges, ColumnMaps columns) {
```

Comment:
Should we keep this overload?

Reply:
I merged the overloads. This also reduces the need to throw exceptions, since we can always build an empty range.